tests/unit/OrgPerson.spec.ts

```diff
@@ -16,238 +13,289 @@ const vuetify = new Vuetify({})
 const store = getVuexStore()
 
 // Events
-const addEditPersonEvent: string = 'changeOrgPerson'
-const removePersonEvent: string = 'removePersonEvent'
-const reassignCompletingPartyEvent: string = 'removeCompletingPartyRole'
-const formResetEvent: string = 'resetEvent'
+const addEditEvent = 'addEdit'
+const removeEvent = 'remove'
+const resetEvent = 'reset'
+const removeCpRoleEvent = 'removeCpRole'
 
 // Input field selectors to test changes to the DOM elements.
-const firstNameSelector: string = '#person__first-name'
```

Comment:
These typings are not needed as the type can be inferred from the assigned value (and some linters will complain that this is redundant).
My rule of thumb: if I hover over the property and can tell its type then I don't specify it.